include/Util/BasicTypes.h

```diff
@@ -251,6 +266,38 @@ class SVFFunction : public SVFValue
         return this->exitBB;
     }
 
+    void getExitBlocksOfLoop(const BasicBlock* bb, Set<const BasicBlock*>& exitbbs) const
+    {
+        if (hasLoopInfo(bb))
+        {
+            const std::vector<const BasicBlock*> blocks = getLoopInfo(bb);
+            if (!blocks.empty())
```

Comment:
blocks should never be empty? Because you always push something into the map `addToBB2LoopMap `?

assert(!blocks.empty() && "no available loop info?")